homeassistant/components/fritz/device_tracker.py

```diff
@@ -145,7 +145,7 @@ def unique_id(self) -> str:
     def ip_address(self) -> str | None:
         """Return the primary ip address of the device."""
         if self._mac:
-            return self._router.devices[self._mac].ip_address
+            return self._router.devices[self._mac].ip_address  # type: ignore[no-any-return]
```

Comment:
These ignores are needed because we haven't typed `router.devices` correctly. We should fix that instead of adding the ignores.